dmriprep/interfaces/images.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Please remove (the shebang should not be defined for this file, and the utf-8 setting should not be necessary anymore)